### ISSUE TITLE
Framework not found linker error with cross platform podfile

### ISSUE THREAD

**Issue by kdawgwilk** [CocoaPods/CocoaPods#4870]
I cannot seem to get cocoapods to work well with cross platform projects.  I currently have project that is a cross platform (tvOS, OSX, iOS) spritekit game with unit/ui test targets as well for each and there always seems to be problems with test targets.

Here is my current Podfile and I have tried getting things to work with both `0.39.0` and `1.0.0.beta.3` neither will work.

``` ruby
source 'https://github.com/CocoaPods/Specs.git'

use_frameworks!

def shared_pods
  pod 'Fabric'
  pod 'Crashlytics'
end

def shared_testing_pods
  pod 'Quick'
  pod 'Nimble'
end

####################################
################ iOS ###############
####################################
target 'iOS' do
  platform :ios, '8.0'
  shared_pods

  target 'iOSTests' do
    inherit! :search_paths
    shared_testing_pods
  end

  target 'iOSUITests' do
    inherit! :search_paths
    shared_testing_pods
  end
end

####################################
################ tvOS ##############
####################################
target 'tvOS' do
  platform :tvos, '9.0'
  shared_pods

  target 'tvOSTests' do
    inherit! :search_paths
    shared_testing_pods
  end

  target 'tvOSUITests' do
    inherit! :search_paths
    shared_testing_pods
  end
end


####################################
################ OSX ###############
####################################
target 'OSX' do
  platform :osx, '10.10'
  shared_pods

  target 'OSXTests' do
    inherit! :search_paths
    shared_testing_pods
  end

  target 'OSXUITests' do
    inherit! :search_paths
    shared_testing_pods
  end
end
```

Its also not very clear what support there is for framework dependencies in UI test targets.

**Comment by PGLongo** [comment]
Same problem for me adding the same pod in **App** target and **AppWatch Extension** I got _No such module_ error on the second target.

I've "resolved" in this way

1. pod deintegrate
1. uninstall current pod version and all the plugins (`gem list --local | grep cocoapods` and then uninstall them)
1. reinstalling _1.0.0.beta.1_
1. pod update

**Comment by lifcio** [comment]
@PGLongo Thanks, that solved it for me.

I also managed to find missing frameworks, but I'm not sure why dod they land there...

Instead of going as usual into 
```
.../your_project_name/Build/Products/...
```
missing frameworks for some reason go into
```
.../your_project_name/Pods/"/Users/username/repated_path_to_your_project/...
e.g. if you keep your project in home directory in Development folder above would be
.../your_project_name/Pods/"/Users/username/Development/your_project_name/Build/Products/...
```
I'm not sure why, but Cocoapods seems to be creating folder named `"` inside `Pods` directory and putting all frameworks in there.

**Comment by mrackwitz** [comment]
I can confirm this issue. This should be fixed by #4146.

**Comment by kdawgwilk** [comment]
I believe this has been resolved in newer versions 👍
